index.Rmd

```diff
@@ -5,7 +5,7 @@ subtitle: An interactive tool for understanding the distribution of invasive ins
 output: html_document
 ---
 
-# Introduction
+# Introduction 
```

Comment:
I really liked your project! I thought it was neat how you added the interactive aspect to your map. I think where you're heading is a great idea, by introducing a more interactive session using Shiny.